state/migration_export.go

```diff
@@ -673,6 +675,7 @@ type addApplicationContext struct {
 }
 
 func (e *exporter) addApplication(ctx addApplicationContext) error {
+	dbModel, err := e.st.Model()
```

Comment:
No need for this, the exporter has a model attribute to get the type

e.model.Type()